napari/utils/shortcuts.py

```diff
@@ -62,14 +69,18 @@
         KeyCode.Digit3,
     ],
     'napari:finish_drawing_shape': [KeyCode.Escape],
-    'napari:reset_active_layer_affine': [
-        KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.KeyR
-    ],
-    'napari:transform_active_layer': [
-        KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.KeyA
-    ],
```

Comment:
hi @brisvag, only noticed this due to merge conflicts with one of my branches, but is there a reason why `reset_active_layer_affine` and `transform_active_layer` were removed? were they moved somewhere I missed?

Reply:
sorry if I missed any comments since this is a massive PR 😅

Reply:
No worries, I lost track of some stuff as well :P These were actions that were quite specific to the old implementation of the `InteractionBox`. `transform_active_layer` is now essentially replaced by the new `activate_<layer_type>_transform_mode` actions. There is no replacement for `reset_active_layer_affine`, but it could (should) re readded.

Reply:
thanks for the clarification! also, if I understand correctly, did you remove the hold to pan/zoom mode in favour of explicitly pressing the corresponding shortcuts?

Reply:
"Hold to pan/zoom" was not a mode, just a `Image` shortcut that temporarily switched mode to `pan_zoom`. I didn't remove it, on the contrary! Now it works on all layers cause it's in `base/_base_key_bindings.py` :P

Reply:
ohhhhh

Reply:
@brisvag revisiting this, it seems that you left in the image hold to pan zoom even though it's now supposed to be on the base layer, was this intentional or should it be removed as well?

Reply:
Oops, definitely a mistake!

Reply:
Created #5586.